mbofui/app/js/controllers.js

```diff
@@ -95,31 +95,29 @@ ocellus.controller('mapController', ['$scope', '$rootScope','$filter', '$timeout
     var endTime = moment($('#endTime').val()).format($rootScope.time_format);
     var postingTime = moment().format($rootScope.time_format);
     var data = {
-      "eventText": $scope.newEventText,
-      "startTime": startTime,
-      "endTime": endTime,
-      "latitude": coords[0],
-      "longitude": coords[1],
-      "altitudeMeters": 266.75274658203125, //this is being removed but the database still expects it
-      "postingTime": postingTime
+      'eventText': $scope.newEventText,
+      'startTime': startTime,
+      'endTime': endTime,
+      'latitude': coords[0],
+      'category':$scope.selected_category.key,
+      'longitude': coords[1],
+      'altitudeMeters': 266.75274658203125, //this is being removed but the database still expects it
```

Comment:
Since it hasn't been decided that altitude will be removed, this comment should be deleted.  It's confusing or, at least, misleading.